master/buildbot/reporters/github.py

```diff
@@ -126,50 +126,53 @@ def send(self, build):
         context = yield props.render(self.context)
 
         sourcestamps = build['buildset'].get('sourcestamps')
-
-        if not sourcestamps or not sourcestamps[0]:
+        if not sourcestamps:
             return
 
-        project = sourcestamps[0]['project']
+        for sourcestamp in sourcestamps:
+            project = sourcestamp['project']
 
-        branch = props['branch']
-        m = re.search(r"refs/pull/([0-9]*)/merge", branch)
-        if m:
-            issue = m.group(1)
-        else:
+            # If branch is None, so is issue
```

Comment:
this line probably does not belong to the change

Reply:
Fixed now.